code/modules/clothing/suits/jobs.dm

```diff
@@ -189,6 +198,8 @@
 	gas_transfer_coefficient = 0.9
 	permeability_coefficient = 0.5
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	style_coverage = COVERS_TORSO|COVERS_UPPER_ARMS
+	style = STYLE_HIGH
```

Comment:
Apron is pretty dull, nothing stylish. Certainly not `STYLE_HIGH`.

Reply:
I was just going off the description, my eye for style is nigh-nonexistent.